README.md

```diff
@@ -29,14 +28,15 @@ Machine, which is required to run Docker on Mac/Windows hosts.
 2. Build the container(s)
 
    ```
-   $ docker-compose build
+   $ docker-compose -f conf/docker/docker-compose.yml build
+   $ npm install
    ```
 
 3. Configure environment variables
 
-   Make a copy of [designsafe.env.sample](designsafe.env.sample) and rename it to
+   Make a copy of [designsafe.sample.env](designsafe.sample.env) and rename it to
```

Comment:
@fnets can you fix the `designsafe.sample.env`, `rabbitmq.sample.env`, and `mysql.sample.env` links?